pandas/core/window.py

```diff
@@ -602,8 +604,8 @@ def validate(self):
         if isinstance(window, (list, tuple, np.ndarray)):
             pass
         elif is_integer(window):
-            if window < 0:
-                raise ValueError("window must be non-negative")
+            if window <= 0:
+                raise ValueError("window must be positive")
```

Comment:
@WillAyd  Combined them as pointed out by @mroeschke

Reply:
My mistake in looking at the git diff I didn't realize they were two different methods. That said, are both required? Doesn't the call to super here raise already in the case where `window == 0`?

Reply:
@WillAyd To be really honest, I don't think that it's required either. Let me remove this